httpx/_transports/wsgi.py

```diff
@@ -62,11 +63,13 @@ def __init__(
         raise_app_exceptions: bool = True,
         script_name: str = "",
         remote_addr: str = "127.0.0.1",
+        log_file: typing.Optional[typing.TextIO] = None,
     ) -> None:
         self.app = app
         self.raise_app_exceptions = raise_app_exceptions
         self.script_name = script_name
         self.remote_addr = remote_addr
+        self.log_file = log_file
```

Comment:
I think `wsgi_errors` would be more consistent here, right?
Loosely matching `script_name` and `remote_addr`.

Reply:
Happy to change it!

Reply:
Donzo